README.md

```diff
@@ -249,6 +249,7 @@ sections = {lualine_a = {'mode'}}
 - `filetype`
 - `hostname`
 - `location` (location in file in line:column format)
+- `searchcount` (number of search matches when hlsearch is active)
```

Comment:
the list is alphabetically sorted :]

Reply:
Oh my! Apologies for that, tried to get this done quickly during a commute. Fixed in the latest commit :)